python/BioSimSpace/Sandpit/Exscientia/Types/_area.py

```diff
@@ -118,6 +118,14 @@ def __init__(self, *args):
     def __mul__(self, other):
         """Multiplication operator."""
 
+        # Handle containers by converting each item in the container to
+        # this type.
+        if isinstance(other, (list, tuple)):
+            container = []
+            for item in other:
+                container.append(self.__mul__(item))
+            return container
```

Comment:
Should the container be of the same type as `other`?

Reply:
Yes, that's a good point. I'll make sure it either returns a list or tuple depending on the original container.